core/ui/legacy/designsystem/src/main/kotlin/app/k9mail/core/ui/legacy/designsystem/atom/icon/Icons.kt

```diff
@@ -0,0 +1,55 @@
+package app.k9mail.core.ui.legacy.designsystem.atom.icon
+
+import app.k9mail.core.ui.legacy.designsystem.R
+
+/**
+ * Icons used in the legacy design system.
+ *
+ * The icons are organized in two types: `Filled` and `Outlined`. Each object contains the icons as drawableRes.
+ */
+object Icons {
+    object Filled {
+        val ARCHIVE = R.drawable.ic_archive
+
+        @JvmStatic
```

Comment:
If we use `@JvmField` instead, this can be referenced as `Icons.Filled.ATTACHMENT_IMAGE` from Java (instead of `Icons.Filled.getATTACHMENT_IMAGE()`).